src/guess/isBalanced.js

```diff
@@ -0,0 +1,26 @@
+function isBalanced(expr) {
```

Comment:
This function is looking sharp! Only bit would be to remove double empty lines and the final `//console.log`